packages/p2p/src/constants/modals.js

```diff
@@ -112,4 +112,7 @@ export const modals = {
     RecommendedModal: React.lazy(() =>
         import(/* webpackChunkName: "recommended-modal" */ 'Components/modal-manager/modals/recommended-modal')
     ),
+    AdExceedsBalanceModal: React.lazy(() =>
+        import(/* webpackChunkName: "recommended-modal" */ 'Components/modal-manager/modals/ad-exceeds-balance-modal')
+    ),
```

Comment:
This should have a different webpackChunkName. Please also import this alphabetically.

Reply:
updated.